### ISSUE TITLE
LibreOffice formula incompatible with quarantine

### ISSUE THREAD

**Issue by cbertelli** [Homebrew/homebrew-cask#51536]
Upgrading libreoffice fails with:
```
Error: Failed to quarantine /Users/user/Library/Caches/Homebrew/downloads/dc9ddd48fec30d8b930c5dcc5d7660263a466a6dcaa6111d9114357afd8cba1b--LibreOffice_6.1.0_MacOS_x86-64.dmg. Here's the reason:
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:7:23: warning: extraneous '_' in parameter: 'string' has no keyword argument name
  mutating func write(_ string: String) { fputs(string, stderr) }
                      ^~
                      
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:5:18: error: use of undeclared type 'TextOutputStream'
struct swifterr: TextOutputStream {
                 ^~~~~~~~~~~~~~~~
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:6:3: warning: declaring a public static var for an internal struct
  public static var stream = swifterr()
  ^~~~~~
  internal
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:10:5: error: use of unresolved identifier 'CommandLine'
if (CommandLine.arguments.count < 4) {
    ^
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:14:58: error: use of unresolved identifier 'CommandLine'
let dataLocationUrl: NSURL = NSURL.init(fileURLWithPath: CommandLine.arguments[1])
                                                         ^
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:21:38: error: use of unresolved identifier 'CommandLine'
  kLSQuarantineDataURLKey as String: CommandLine.arguments[2],
                                     ^
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:22:40: error: use of unresolved identifier 'CommandLine'
  kLSQuarantineOriginURLKey as String: CommandLine.arguments[3]
                                       ^
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:27:8: error: consecutive statements on a line must be separated by ';'
    try dataLocationUrl.setResourceValue(
       ^
       ;
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:31:4: error: expected 'while' in 'do-while' loop
  }
   ^
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:32:9: error: braced block of statements is an unused closure
  catch {
        ^
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/utils/quarantine.swift:32:9: error: type of expression is ambiguous without more context
  catch {
        ^
```

**Comment by vitorgalvao** [comment]
> [If you ignore this guide, your issue may be closed without review](https://github.com/homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).

Please open a new issue and fill all the information requested. We’re interested in this bug report, but we need the information we ask for.

**Comment by ioben** [comment]
I had this issue when trying to install any cask (specifically `java8` first, then a few others as spot checks). The `brew doctor` and `brew cask doctor` commands didn't return any errors.  The only way I was able to fix it was by upgrading from 10.11 to latest OSX (10.13.6), and upgrading XCode to the latest version (9.4.1).
